packages/main/src/plugin/kubernetes-context-state.ts

```diff
@@ -297,11 +364,29 @@ export class ContextsManager {
 
     const ns = context.namespace ?? 'default';
     return {
-      podInformer: this.createPodInformer(kc, ns, context),
-      deploymentInformer: this.createDeploymentInformer(kc, ns, context),
+      pods: this.createPodInformer(kc, ns, context),
+      deployments: this.createDeploymentInformer(kc, ns, context),
     };
   }
 
+  startResourceInformer(contextName: string, resourceName: ResourceName): void {
+    const context = this.kubeConfig.contexts.find(c => c.name === contextName);
+    if (!context) {
+      throw new Error('context not found');
+    }
+    const ns = context.namespace ?? 'default';
+    let informer: Informer<KubernetesObject> & ObjectCache<KubernetesObject>;
+    switch (resourceName) {
+      case 'services':
+        informer = this.createServiceInformer(this.kubeConfig, ns, context);
+        break;
+      default:
+        console.debug(`trying to start informer for resource ${resourceName} which is not supported`);
```

Comment:
need the contextName displayed in the log I would say

Also informer looks like too technical/internal if end user look the message IMHO he would expect what was the intent

`Unable to monitor/watch/whatever resource ${resourceName} for context ${contextName} as it is not supported`